content/backend/graphql-js/5-connecting-server-and-database.md

```diff
@@ -192,7 +192,6 @@ const server = new GraphQLServer({
     db: new Prisma({
       typeDefs: 'src/generated/prisma.graphql',
       endpoint: 'https://eu1.prisma.sh/public-graytracker-771/hackernews-node/dev',
-      secret: 'mysecret123',
```

Comment:
Again, same here - if we have the `secret` in `prisma.yml`, we need it in the constructor.